# Courses
## R courses
### 2020
#### ES7023 & ES0002, supervised by David Lallemant: Fundamentals of Data Science for Earth and Environmental Systems Science, ASE, Nanyang Technological University
* [Session 01 - 2020-01-22 - Introduction to R](https://vaulot.github.io/course-ntu-data-science-2020/R-session-01-intro.html)
* [Session 02 - 2020-01-24 - R markdown](https://vaulot.github.io/course-ntu-data-science-2020/R-session-02-markdown.html)
* [Session 03 - 2020-01-24 - Git](https://vaulot.github.io/course-ntu-data-science-2020/R-session-03-Git.html)
* [Session 04 - 2020-02-07 - Git](https://vaulot.github.io/course-ntu-data-science-2020/R-session-04-data_wrangling.html)
* [Session 05 - 2020-02-14 - Git](https://vaulot.github.io/course-ntu-data-science-2020/R-session-05-data_visualization.html)

#### Experimental Design and Analysis, supervised by Eleanor Slade, ASE, Nanyang Technological University
* [Session 01 - 2020-01-21 - Introduction to R](https://vaulot.github.io/course-ntu-experimental-design-2020/R-session-01-intro.html)
* [Session 02 - 2020-01-23 - Data visualization](https://vaulot.github.io/course-ntu-experimental-design-2020/R-session-02-data_visualization.html)
* [Session 03 - 2020-01-23 - R markdown](https://vaulot.github.io/course-ntu-experimental-design-2020/R-session-03-markdown.html)

### 2019
#### ES7023 & ES0002, supervised by David Lallemant - 2019: Fundamentals of Data Science for Earth and Environmental Systems Science, ASE, Nanyang Technological University
* [Session 01 - 2019-01-17 - Introduction to R](https://vaulot.github.io/course-ntu-data-science-2019/R-session-01-intro.html)
* [Session 02 - 2019-01-24 - R markdown](https://vaulot.github.io/course-ntu-data-science-2019/R-session-02-markdown.html)
* [Session 03 - 2019-01-31 - Data wrangling](https://vaulot.github.iocourse-ntu-data-science-2019/R-session-03-data_wrangling.html)
* [Session 04 - 2019-02-07 - Data visualization](https://vaulot.github.io/course-ntu-data-science-2019/R-session-04-data_visualization.html)
* [Session 05 - 2019-03-15 - Mapping](https://vaulot.github.io/course-ntu-data-science-2019/R-session-05-mapping.html)
* [Session 06 - 2019-04-18 - Git](https://vaulot.github.io/course-ntu-data-science-2019/R-session-06-Git.html)

# Tutorials
## R
* [R introduction](https://vaulot.github.io/tutorials/R_introduction_tutorial.html)
* [Phyloseq](https://vaulot.github.io/tutorials/Phyloseq_tutorial.html)
* [Plotting culture data](https://vaulot.github.io/tutorials/R_tutorial_cultures.html)

## Metabarcoding
* [Metabarcoding analysis with dada2](https://vaulot.github.io/tutorials/R_dada2_tutorial.html)
* [Compare analysis with mothur vs. dada2](https://vaulot.github.io/tutorials/R_dada2_vs_mothur.html)

# Metabarcoding

## BIOSOPE 2013
* [03 - Sequence processing](https://vaulot.github.io/metabarcoding/biosope/03-BIOSOPE_sequence_processing.html)
* [04 - Mothur vs Dada2](https://vaulot.github.io/metabarcoding/biosope/04-BIOSOPE_method_comparison.html)
* [05 - BLAST](https://vaulot.github.io/metabarcoding/biosope/05-BIOSOPE_blast.html)

## NZ 2017
* [06 - Create phyloseq](https://vaulot.github.io/metabarcoding/nz_2017/06-NZ_2017_phyloseq_create.html)
* [07 - BLAST](https://vaulot.github.io/metabarcoding/nz_2017/07-NZ_2017_blast.html)
* [08 - Phyloseq examples](https://vaulot.github.io/metabarcoding/nz_2017/08-NZ_2017_phyloseq_plots.html)

## Singapore
* [Phyloseq analysis](https://vaulot.github.io/metabarcoding/singapore/Metabarcoding_Singapore_phyloseq)
* [Eukaryote synthesis](https://vaulot.github.io/metabarcoding/singapore/Metabarcoding_Singapore_eukaryotes)
* [Eukaryote analysis 97%](https://vaulot.github.io/metabarcoding/singapore/Metabarcoding_Singapore_97.html)
* [Eukaryote analysis 98%](https://vaulot.github.io/metabarcoding/singapore/Metabarcoding_Singapore_98.html)
* [Eukaryote analysis ASV dada2](https://vaulot.github.io/metabarcoding/singapore/Metabarcoding_Singapore_ASV)

## Antarctica 2015 - Sorted samples
* [Dada2](https://vaulot.github.io/metabarcoding/antarctica_2015/Antar_dada2.html)

# Cruises
* [Ilhas cruises 1 and 2 - Brazil 2017](https://vaulot.github.io/cruises/Ilhas_01.html)
* [Green Edge - 2015-2016 - Ice Camp](https://vaulot.github.io/cruises/Green_Edge_Ice_Camp.html)

# Metagenomes
* [Subrata fish metagenomes - 2018](https://vaulot.github.io/metagenomes/Subrata_fish_metagenome_2018.html)

# Databases

## PR2
* [PR2 statistics](https://vaulot.github.io/pr2/PR2_analysis.html)
* [PR2 update 4.11 - Management](https://vaulot.github.io/pr2/PR2_update_4.11.0_Misc.html)
* [PR2 update 4.11 - Chloropicophyceae](https://vaulot.github.io/pr2/PR2_update_4.11.0_Chloropicophyceae.html)
* [PR2 update 4.11 - Mamiellophyceae](https://vaulot.github.io/pr2/PR2_update_4.11.0_Mamiellophyceae.html)
* [PR2 update 4.11 - Ciliophora](https://vaulot.github.io/pr2/PR2_update_4.11.0_EukRef_Ciliates.html)
* [PR2 update 4.10](https://vaulot.github.io/pr2/PR2_update_4.10.0_pr2_original.html)
* [PR2 update 4.9](https://vaulot.github.io/pr2/PR2_update_4.9.0_DinoRef.html)

## PR2
* [PR2 V4-V9 primer analysis](https://vaulot.github.io/pr2/PR2_Primers.html)
* [PR2 primer analysis - set 32 - Fede Universal](https://vaulot.github.io/pr2/PR2_Primer_Set_32.html)
* [PR2 primer analysis - set 83 - Wenshu V4](https://vaulot.github.io/pr2/PR2_Primer_Set_83.html)
## RCC
* [RCC statistics](https://vaulot.github.io/rcc/RCC_statistics.html)
* [RCC extract data](https://vaulot.github.io/rcc/RCC_custom_lists.html)
* [Submit sequences to Genbank](https://vaulot.github.io/rcc/RCC-genbank-submission.html)
* [Submit sequences to Genbank - pdf version](https://vaulot.github.io/rcc/RCC-genbank-submission.pdf)

# Papers
* [Distribution of Bolidophyceae](https://vaulot.github.io/papers/Bolido_review.html)
* [Distribution of Malina RCC2288](https://vaulot.github.io/papers/RCC2288.html)
* [Distribution of Chlorophyta at OSD stations](https://vaulot.github.io/papers/OSD_Chlorophyta.html)
* [Distribution of Mamiellophyceae at OSD stations](https://vaulot.github.io/papers/OSD_Mamiello.html)
* [Distribution of Pelagophyceae at OSD stations](https://vaulot.github.io/papers/Pelago_metabarcodes_version_2.0.html)
